### ISSUE TITLE
figure out what's happening with creating dup tree codes in forestplots import

### ISSUE THREAD

**Issue by ashenkin** [OxfordEcosystemsLab/gemdata#30]
e.g.: [16:44:05] [ERROR] Trying to import a duplicate tree code for DUP585 in plot TAM-06
New fp id is 56720 vs existing fp id 56721
Saving with tree code DUP585

**Comment by ashenkin** [comment]
Thanks C,

Yeah, "roots" is written in the tree notes column.  And unfortunately, the way they've tagged them has been to append a ".r" to the subplot number.  What does this mean?  Are they ramets (i.e. clones from roots)?

If ramets, they don't keep track of the ramet number anywhere; they just rely on their database-given TreeID to keep track of individual ramets (lord knows how they keep track of these in the field).

**Comment by cecilegirardin** [comment]
No, they are multiple big roots from one tree. I will mail William to
confirm.

On 19 February 2015 at 11:36, Alexander Shenkin <notifications@github.com>
wrote:

> Thanks C,
>
> Yeah, "roots" is written in the tree notes column. And unfortunately, the
> way they've tagged them has been to append a ".r" to the subplot number.
> What does this mean? Are they ramets (i.e. clones from roots)?
>
> If ramets, they don't keep track of the ramet number anywhere; they just
> rely on their database-given TreeID to keep track of individual ramets
> (lord knows how they keep track of these in the field).
>
> On 2/19/2015 11:29 AM, Cécile Girardin wrote:
>
> Hi guys,
> There seem to be some duplicate tree numbers in Forestplots (shouldn't be,
> but I keep on finding a few).
> Just one thing to be careful about: make sure they are not duplicates
> because they are roots rather than stems. It should be indicated in one of
> the columns. William and Gaby came up with a code for roots (possibly just
> an R in front of tree number?).
> Hope this helps,
> Cécile
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OxfordEcosystemsLab/gemdata/issues/30#issuecomment-75038053>
> .
>



-- 
Dr. Cécile Girardin
Tel: (+ 44) 07980 011 347

gem.tropicalforests.ox.ac.uk
cecilegirardin.com
oxfordbiochar.com
instagram.com/cecilegirardin/

**Comment by ashenkin** [comment]
so, they're tracking root?  or... some of the multiples are for a ficus... so these are multiple strangling roots?  i imagine that must be it...

**Comment by cecilegirardin** [comment]
I mailed William to confirm.
About Gaby- not a clue. You could try calling her at the office?


On 19 February 2015 at 11:48, Alexander Shenkin <notifications@github.com>
wrote:

> so, they're tracking root? or... some of the multiples are for a ficus...
> so these are multiple strangling roots? i imagine that must be it...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/OxfordEcosystemsLab/gemdata/issues/30#issuecomment-75039474>
> .
>



-- 
Dr. Cécile Girardin
Tel: (+ 44) 07980 011 347

gem.tropicalforests.ox.ac.uk
cecilegirardin.com
oxfordbiochar.com
instagram.com/cecilegirardin/
